lib/src/widgets/custom_image_crop_widget.dart

```diff
@@ -37,22 +37,30 @@ class CustomImageCrop extends StatefulWidget {
   /// we've provided two default painters as inspiration
   /// `DottedCropPathPainter.drawPath` and
   /// `SolidCropPathPainter.drawPath`
-  const CustomImageCrop({
+  CustomImageCrop({
     required this.image,
     required this.cropController,
     this.overlayColor = const Color.fromRGBO(0, 0, 0, 0.5),
     this.backgroundColor = Colors.white,
     this.shape = CustomCropShape.Circle,
     this.cropPercentage = 0.8,
     this.drawPath = DottedCropPathPainter.drawPath,
+    Paint? imagePaintDuringCrop,
     Key? key,
-  }) : super(key: key);
+  }) : super(key: key) {
+    if (imagePaintDuringCrop == null) {
```

Comment:
this.imagePaintDuringCrop =  imagePaintDurtingCrop ?? ...

Reply:
Yes, I thought so to, but we do `..filterQuality = FilterQuality.high`
The following code snippet illustrates the issue:
```dart
  final List? list = [];
  final list2 = list ?? []..add('');
  print(list2.length); // 1
```

Reply:
Add parentesis